rfcs/2021-11-whcm-support.md

```diff
@@ -0,0 +1,108 @@
+
+`[forced-colors](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/forced-colors)` is a new media query supported by Chrome, Edge, and Firefox. It supports `none` and `active` as values. It should also be noted that this media feature spec is actively being worked on.
+
+[Windows recommends using both media queries](https://blogs.windows.com/msedgedev/2020/09/17/styling-for-windows-high-contrast-with-new-standards-for-forced-colors/) (aka ["The Frankenquery"](https://adrianroselli.com/2021/02/whcm-and-system-colors.html#Frankenquery)) to guarantee adequate support for those using WHCM. This would look something like:
+
+```css
+@media (-ms-high-contrast: active), (forced-colors: active) {
+  /* CSS overrides here */
+}
+```
+
+While our design system no longer technically supports IE (as [it falls below our 2% support threshold](https://analytics.usa.gov/health-human-services/)), a good percentage of those who rely on WHCM continue to use it. And because the effort to support IE is low, I agree with Microsoft's recommendation - just be certain your system color keywords work across both queries and the cascade doesn't impact how those styles render (meaning, test the changes). If more targeted styles are needed between IE and Edge, this query may need to be split.
+
+We have access to a SauceLabs account and they have [good documentation for how to do local testing](https://docs.saucelabs.com/secure-connections/sauce-connect/quickstart/).
+
+**And to note, before any work begins:**
+
+It's not the intention of this work to create a separate design for WHCM users. If semantic HTML is used correctly, there should be very little need to overwrite styles. The overall recommendation is to use a light touch when applying fixes. As [Adrian Roselli writes](https://adrianroselli.com/2021/02/whcm-and-system-colors.html#FQs):
+
+> You should only ever use these feature queries to spackle over gaps (found in most libraries / frameworks) or enhance an experience that may be lacking.
+
+And as [MSN says](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/forced-colors#accessibility_concerns):
+
+> [...] its intended usage is to make small tweaks to improve usability or legibility when the default application of forced colors does not work well for a given portion of a page.
+
+I believe the outages identified above (with maybe the exception of Buttons) fall into this definition.
+
+### Benefits
+
+- We get to be Good Guy Gregs by supporting a widely used accessibility feature
+- Our design system delivers a consistent experience regardless of user setting
+- Current unusable components will be usable to our users and the end users
```

Comment:
I like this bullet point as a way to prioritize HCM work. Take the components that are flat-out unusable, sort by prevalence, then chip away at them. Things that technically work, but could benefit from a small tweak or two follow afterwards.